<!DOCTYPE html>
<html>
    <head>
        <script data-require="jquery@*" data-semver="2.0.3" src="http://code.jquery.com/jquery-2.0.3.min.js"></script>
        <script src="demo-settings.js"></script>
        <title>Genomic Medicine Assistant Demo</title>
        <link rel='stylesheet' href='main.css'></link>
        <link rel='stylesheet' href='report.css'></link>
        <link rel='stylesheet' href='inbox.css'></link>
        <script type="text/javascript" src="jquery-latest.js"></script>
        <script src="light_box.js"></script>
        <script>
			
				$(document).ready(function(){
		    		$(".div2").hide();
		        	document.getElementById("selected_one").className = document.getElementById("selected_one").className + " selected";  // this adds the selected class
				    $(".link1, .link2").bind("click", function () {
			    	  $(".div1, .div2").hide();        
		      		  if ($(this).attr("class") == "link1")
		      		  {
		        		$(".div1").show();
		        		document.getElementById("selected_one").className = document.getElementById("selected_one").className + " selected";  // this adds the selected class
						document.getElementById("selected_two").className = document.getElementById("selected_two").className.replace(" selected", ""); // this removes the selected class
		      		  }
		      		  else 
		      		  { 
		        		$(".div2").show();
						document.getElementById("selected_two").className = document.getElementById("selected_two").className + " selected";  // this adds the selected class
						document.getElementById("selected_one").className = document.getElementById("selected_one").className.replace(" selected", ""); // this removes the selected class
		      		  }
		    		});
				});
			</script>
		</head>
		<body>
			<div id="action_box">
				<!-- Popup Div Starts Here -->
				<div id="action_items">
					<button onclick ="action_hide()" id="close">X</button>
					<h2>Actions:</h2>
						<form action="">
							<input type="checkbox" name="action" value="email_patient">Email letter to patient<br>
							<input type="checkbox" name="action" value="email_patient">Email patient resource to patient<br>
							<button onclick="div_hide()" id="send">Send</button>
						</form>
				</div>
				<!-- Popup Div Ends Here -->
			</div>			
			<div id="evidence_box" >
				<!-- Popup Evidence Div Starts Here -->
				<div id="evidence_items" >
					<button onclick ="evidence_hide()" id="close">X</button>
                    <iframe id="reference_iframe" src="./popup/scn5a-evidence.html" width="100%" height="100%">
                    </iframe>
				</div>
				<!-- Popup Evidence Div Ends Here -->
			</div>			
			<div id="reference_box">
				<!-- Popup Patient Resource Div Starts Here -->
				<div id="reference_items">
					<button onclick ="reference_hide()" id="close">X</button>
                    <iframe id="reference_iframe" src="https://ghr.nlm.nih.gov/condition/brugada-syndrome" width="100%" height="100%"></iframe>
				</div>
				<!-- Popup Patient Resource Div Ends Here -->
			</div>			
			<div id="information_box">
				<!-- Popup Gene Information Div Starts Here -->
				<div id="information_items">
					<button onclick ="information_hide()" id="close">X</button>
                    <iframe id="reference_iframe" src="https://ghr.nlm.nih.gov/gene/SCN5A" width="100%" height="100%"></iframe>
				</div>
				<!-- Popup Gene Information Div Ends Here -->
			</div>
            <div id="variant_box">
                <!-- Popup Variant Information Div Starts Here -->
                <div id="variant_items">
                    <button onclick ="variant_hide()" id="close">X</button>
                    <iframe id="reference_iframe" src="http://www.ncbi.nlm.nih.gov/clinvar/variation/201596" width="100%" height="100%"></iframe>
                </div>
                <!-- Popup Variant Information Div Ends Here -->
            </div>
            
            <p>Cutting E, Banchero M, Beitelshees AL, Cimino JJ, Del Fiol G, Gurses AP, Hoffman MA, Jeng LJ, Kawamoto K, Kelemen M, Pincus HA, Shuldiner AR, Williams MS, Pollin TI, Overby CL. <b>User-centered design of multi-gene sequencing panel reports for clinicians.</b> Journal of Biomedical Informatics. 2016 Jul 14.</p>
                        
			<div id='app_title'>
				<h1>Genomic Medicine Assistant</h1>
			</div>
			<div id="main">
				<h1><span style="color:#002D72;">Pharmacogenomic Sequencing Panel Report </span></h1>
				<div id="results">
					<ul>
						<li id="selected_one"><a href="#" class="link1">Results</a></li>
						<li id="selected_two"><a href="#" class="link2">Test Information</a></li>
					</ul>
					<div class="div1" >
                    
 <!-- RESULTS TABLE -->
                        <table>
                            <tr>
                                <th>Patient</th>
                                <td>Mouse, Mickey</td>
                                <th>Specimen Type:</th>
                                <td>Whole Blood</td>
                            </tr>
                            <tr>
                                <th>Patient DOB</th>
                                <td>06/15/1945</td>
                                <th>Collection Date:</th>
                                <td>04/22/2015</td>
                            </tr>
                       
                            <tr>
                                <th>Ordering Provider</th>
                                <td>Dr. Seuss</td>
                                <th>Received Date:</th>
                                <td>04/22/2015</td>
                            </tr>
                            <tr>
                                <th>Institution:</th>
                                <td>Disney Medical Center</td>
                                <th>Lab Accession No.</th>
                                <td>600254</td>
                            </tr>
                        </table>
						<div id="results_info_div">
                        
 <!-- RESULTS INFO -->
 
                        <p>This individual has results from a multi-gene sequencing panel that is being used to optimize drug therapy.  A sequence variant in a gene, unrelated to the primary indication for testing, was identified by this assay as an incidental finding and is reported below.</p>

						</div>
						<div id="variants_div" style="overflow-x:auto;">
                        
<!-- VARIANTS TABLE -->
                        
							<h3>Variants</h3>
							<table>
								<tr>
									<th>Gene</th>
									<th>Variant</th>
									<th>Clinical Significance</th>
								</tr>
								<tr>
									<td><i>SCN5A</i>
									<button id="information" onclick="information_show()">i</button>
                                 
									</td>
                                    <td>c.5830C>T (p.Arg1944Ter); heterozygous
                                    <button id="variant" onclick="variant_show()">i</button>
                                    </td>
									<td>Pathogenic variant in gene associated with Brugada Syndrome</td>
								</tr>
							</table>
						</div>
						<div id="interpretation">
							<button id="evidence" onclick="evidence_show()">Evidence</button>
                            
<!-- INTERPRETATION -->
                            
							<h3>Interpretation Summmary</h3>

                           <p>This individual is heterozygous for the p.Arg1944* truncating variant in the SCN5A gene.</p>
                            <p>Brugada Syndrome 1 is an autosomal dominant cardiac conduction abnormality resulting in high risk for ventricular arrhythmia and sudden cardia death.[1,2] 15-30% of individuals with Brugada Syndrome 1 possess pathogenic variants in the SCN5A gene.[3]</p>
                            <p>Although the Arg1944Stop mutation has not been reported previously as a disease-causing mutation or as a benign polymorphism to our knowledge, it is predicted to cause loss of normal protein function through protein truncation.[4]</p>
                            <p>Brugada Syndrome patients with truncation mutations may have a more severe phenotype characterized as a higher propensity for syncope and prevalence of sudden cardiac death among young first-degree relatives compared to those patients with missense variants functionally characterized with <90% peak sodium current reduction[5]</p>

    </div>
						<div id="recommendations">
							<button id="patient_resource" onclick="reference_show(), setHeight()">Patient Resource</button>
                            
<!-- RECOMMENDATIONS -->
                            
							<h3>Recommendations</h3>
                            
                            
                            <p>Diagnosis is based on clinical findings. It is recommended to evaluate clinical history, family history, signs and symptoms in order to determine a diagnosis.</p>
                            <p> Upon clinical evaluation, if you believe this patient has a genetic condition, evaluation by a genetic specialist to determine the need for additional genetic testing may be warranted. </p>
							<p>Contact the Pharmacogenomic Counseling Service for additional information at 123-456-7890</p>
                            <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">
                                Signed by:</span></b> Walt Disney, PhD, FACMG</p>
                            <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">Report date:</span></b>  June 2, 2015</p>
						</div>
					</div>
					<div class="div2" style="overflow-x:auto;">
                            
<!-- RESULTS TABLE (Test Info tab) -->
                            

                            
                            <table>
                                <tr>
                                    <th>Patient</th>
                                    <td>Mouse, Mickey</td>
                                    <th>Specimen Type:</th>
                                    <td>Whole Blood</td>
                                </tr>
                                <tr>
                                    <th>Patient DOB</th>
                                    <td>06/15/1945</td>
                                    <th>Collection Date:</th>
                                    <td>04/22/2015</td>
                                </tr>
                                
                                <tr>
                                    <th>Ordering Provider</th>
                                    <td>Dr. Seuss</td>
                                    <th>Received Date:</th>
                                    <td>04/22/2015</td>
                                </tr>
                                <tr>
                                    <th>Institution:</th>
                                    <td>Disney Medical Center</td>
                                    <th>Lab Accession No.</th>
                                    <td>600254</td>
                                </tr>
                            </table>
                            <div id="results_info_div">

                        
<!-- VARIANTS TESTED (Test Info tab) -->
                        
                        <h3>Test Methods and Limitations</h3>
                        
                        <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">
                            Pharmacogenomic Sequencing Panel</span></b></p>
                        <p>MPS was performed to screen 84 pharmacogenes for DNA sequence variants relevant for drug therapy decisions. Sequence variations in a set of genes related to 'actionable genes' were also identified by this assay as incidental findings. These actionable genes were chosen based on their association with adult onset conditions for which there is a recommended screening and/or treatment.</p>
                        
                        <p>Only pathogenic variants are reported for incidental findings - variants of uncertain significance are not returned. Positive findings were validated to clinical standards in a CLIA compliant laboratory; however, the performance characteristics of MPS for negative results (the absence of mutations) were not validated to clinical standards in a CLIA compliant laboratory. This laboratory test was developed and its performance characteristics determined by the University of Maryland Translational Genomics Laboratory (CLIA-certified). Consistent with laboratory-developed tests, this test has not been cleared or approved by the U.S. Food and Drug Administration.</p>
                        
                        <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">Procedure:</span></b></p>
                        <p>I. Pharmacogenomic Sequencing Panel<br>
                        Genomic DNA was extracted from blood using standard procedures. A library of DNA fragments was constructured and enriched for protein and RNA coding portions of the human genome using the PGRNseq custom capture reagent. Sequence captured from each gene includes the complete coding regions plus 2 kb upstream and 1kb downstream to assess variation within nearby regulatory regions. Paired-end sequencing of the enriched library was performed using standard TruSeq v3.0 (Illumina) chemistry on a HiSeq 2000 (Illumina) sequencer. Resulting sequences were aligned to the human genome reference (hg19) using the Burrows_Wheeler Aligner (BWA) and variants identified with the Genome Analysis Tool Kit (GATK). A modified version of the SeattleSeq tool was used to annotate variants found within a defined set of actionable genes.</p>
                        <p>II. Validation of Sequence Variants (CLIA-accredited)<br>
                        Primers based on the hg19 (February 2009) version of the human genome sequence, were used to amplify standard dye-terminator chemistry.</p>
                        
                         <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">Limitations</span></b></p>
                         <p>1. This assay does not detect large deletions or duplications and has limited ability to identify small insertions and deletions. This test also has limited ability to detect mosaicism.</p>
                         <p>2. This assay does not detect variants located: a) outside of 84 pharmacogenes, b) in regions of insuffienct coverage, c) in regions containing paralogous genes or pseudogenes, or d) where the reference genome is inaccurate or contains gaps and insertions.</p>
                         <p>3. Variants of uncertain significance are not reported.</p>
                         <p>4. Genes not associated with treatable genetic conditions at the time this test was performed were not analyzed.</p>

                        
                        <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">DISCLAIMER</span></b></p>
                        <p>This test was developed and its performance characteristics determined by the University of Schrek Genomics Laboratory. It has not been cleared or approved by the FDA.  This laboratory is certified under the Clinical Laboratory Improvement Amendments of 1988 (CLIA-88) as qualified to perform high complexity clinical laboratory testing.  This test is for clinical purposes.  It should not be regarded as investigational or for research.</p>
                       
                       <p><b><span style="color:#002D72; font-family:'Gill Sans', 'Gill Sans MT', 'Myriad Pro', 'DejaVu Sans 		Condensed', Helvetica, Arial, sans-serif;">
                           Testing Performed at: </span></b><br>
                           University of Schrek Genomics Laboratory<br>
                           12345 Fantasy Road<br>
                           Orlando, FL 56789<br>
                           123-234-3456</p>

						
                            </div>
                    </div>
                </div>
                <input type="submit" name="selection" value="Return to inbox" onclick="window.location='inbox.html';"></><br>
                    </div>
            <div id="footer-row1">
                <div id="footer-bg">
                    <div id="footer" class="container_24">
                        <div class="copyright">
                            <p>&COPY;2016</p>
                            <p>While the investigators in the study have made every effort to ensure that the information presented is consistent with current practices, all information presented was prepared for simulated conditions and may not reflect all aspects of standard medical care.</p>
                        </div>
                    </div>
                </div>
            </div>
            
  			<script src="https://cdn.rawgit.com/smart-on-fhir/client-js/v0.0.3/dist/fhir-client.js"></script>
		  	<script src="/static/./js/get-data.js"></script>
		</body>
	</html>